KsApi/models/ProjectState.swift

```diff
@@ -0,0 +1,12 @@
+import Argo
```

Comment:
Do you need this Argo import?

Reply:
I think I needed it to disambiguate Decodable, will check.